xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.cpp

```diff
@@ -674,12 +675,17 @@ DEBUG_INFO_VIDEO CRendererBase::GetDebugInfo(int idx)
 
   const char* px = av_get_pix_fmt_name(rb->pixelFormat);
   const char* pr = av_color_primaries_name(rb->primaries);
+  const char* opr = av_color_primaries_name(rb->m_originalPrimaries);
   const char* tr = av_color_transfer_name(rb->color_transfer);
 
   const std::string pixel = px ? px : "unknown";
-  const std::string prim = pr ? pr : "unknown";
+  std::string prim = pr ? pr : "unknown";
+  const std::string oprim = opr ? opr : "unknown";
```

Comment:
Maybe:

```c++
  const std::string oprim = opr ? opr : "unknown";
  const std::string prim =
      (rb->primaries != rb->m_originalPrimaries) ? (oprim + ">" + pr) : (pr ? pr : "unknown");
```

Seems "better" because not needs remove `const` and avoids extra `if` although the resulting code is not exactly intuitive.

Reply:
Sure but slightly revised.